federation/backends/collections.md

```diff
@@ -1,6 +1,27 @@
 # Collections
 This page describes requirements on STAC collection metadata for backend providers in the openEO platform federation. These requirements should be considered an addition to what is already required by the [openEO API](https://api.openeo.org/#tag/EO-Data-Discovery/operation/describe-collection).
 
+## Collection availability
+
+The collections are a key asset of the platform, and users need to know what they can expect from a certain collection.
+The federation relies on openEO collection metadata following the STAC specification to communicate this towards the user. Backend providers
+need to comply with these requirements for their collection:
+
+The 'experimental' flag from https://stac-extensions.github.io/version/v1.1.0/schema.json needs to be set on experimental collections. 
+A collection can be experimental either because there are issues with the actual data or catalog, or because of backend specific issues that make the use of the collection unstable. 
```

Comment:
```suggestion
A collection can be experimental either because there are issues with the actual data or catalog, or because of backend specific issues that make the use of the collection unstable (e.g., the collection was produced within a project and it's not clear how long it will be accessible through the backend). 
```

Reply:
I somehow understood backend specific issues differently (but after the explanation above it's now clear). As you agreed to add some "min. validity" time for the non experimental collections, I think we can omit this here.